tools/RAiDER/aria/prepFromARIA.py

```diff
@@ -0,0 +1,212 @@
+# ~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
+#
+# Author: Jeremy Maurer, Brett Buzzanga
+# Copyright 2022, by the California Institute of Technology. ALL RIGHTS
+# RESERVED. United States Government Sponsorship acknowledged.
+#
+# ~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
+import os
+from datetime import datetime
+import numpy as np
+import xarray as xr
+import rasterio
+import yaml
+import RAiDER
+from RAiDER.utilFcns import rio_open, writeArrayToRaster
+from RAiDER.logger import logger
+
+""" Should be refactored into a class that takes filename as input """
+## ---------------------------------------------------- Prepare Input from GUNW
+def makeLatLonGrid(f:str, out_dir:str):
+# def makeLatLonGrid(f:str, reg, out_dir):
+    ds0  = xr.open_dataset(f, group='science/grids/data')
+
+    Lat, Lon  = np.meshgrid(ds0.latitude.data, ds0.longitude.data)
+
+    da_lat = xr.DataArray(Lat.T, coords=[Lon[0, :], Lat[:, 0]], dims='lon lat'.split())
+    da_lon = xr.DataArray(Lon.T, coords=[Lon[0, :], Lat[:, 0]], dims='lon lat'.split())
+    # dst_lat = op.join(out_dir, f'lat_{reg}.geo')
+    # dst_lon = op.join(out_dir, f'lon_{reg}.geo')
+
+    dst_lat = os.path.join(out_dir, 'lat.geo')
+    dst_lon = os.path.join(out_dir, 'lon.geo')
+    da_lat.to_netcdf(dst_lat)
+    da_lon.to_netcdf(dst_lon)
+    logger.debug('Wrote: %s', dst_lat)
+    logger.debug('Wrote: %s', dst_lon)
+    return dst_lat, dst_lon
+
+
+def getHeights(f:str):
+    ds =  xr.open_dataset(f, group='science/grids/imagingGeometry')
+    hgts = ds.heightsMeta.data.tolist()
+    return hgts
+
+
+def makeLOSFile(f:str, filename:str):
+    """ Create line-of-sight file from ARIA azimuth and incidence layers """
+
+    group   = 'science/grids/imagingGeometry'
+    azFile  = os.path.join(f'NETCDF:"{f}":{group}/azimuthAngle')
+    incFile = os.path.join(f'NETCDF:"{f}":{group}/incidenceAngle')
```

Comment:
Can we pass somewhere at the top the lcoation of the az, inc etc.
For now if you use the orbit this should not be needed.

Reply:
We can but I think they should just be removed and we always use orbit file

Reply:
Yes this should not be needed with orbit